snowflake_test.go

```diff
@@ -0,0 +1,368 @@
+package snowflake
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/vault/sdk/database/dbplugin/v5"
+	dbtesting "github.com/hashicorp/vault/sdk/database/dbplugin/v5/testing"
+	"github.com/snowflakedb/gosnowflake"
+	_ "github.com/snowflakedb/gosnowflake"
+)
+
+const (
+	envVarSnowflakeAccount  = "SNOWFLAKE_ACCOUNT"
+	envVarSnowflakeUser     = "SNOWFLAKE_USER"
+	envVarSnowflakePassword = "SNOWFLAKE_PASSWORD"
+	envVarSnowflakeDatabase = "SNOWFLAKE_DATABASE"
+	envVarSnowflakeSchema   = "SNOWFLAKE_SCHEMA"
+
+	envVarRunAccTests = "VAULT_ACC"
+)
+
+var runAcceptanceTests = os.Getenv(envVarRunAccTests) != ""
+
+func connectToInstance(t *testing.T) string {
```

Comment:
Nit: `connURL(...)` might be a more fitting name, since this func doesn't actually establish a connection.

Reply:
Oops, forgot to update that one. This used to connect to the instance, but it seemed more useful to do that elsewhere.